ocs_ci/deployment/helpers/prechecks.py

```diff
@@ -0,0 +1,162 @@
+"""
+This module contains the pre-checks needed for deployment.
+Some of the common pre-checks are memory, CPU and storage. If the minimum
+requirements are not satisfied in the target environment, there is no
+need to proceed with deployment.
+This module is mainly intended for on-prem platforms.
+"""
+
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import MIN_STORAGE_FOR_DATASTORE
+from ocs_ci.ocs.exceptions import StorageNotSufficientException
+from ocs_ci.utility.vsphere import VSPHERE as VSPHEREUtil
+
+logger = logging.getLogger(__name__)
+
+
+class PRECHECKS(object):
```

Comment:
Class names should normally use the CapWords convention.

```suggestion
class PreChecks(object):
```

Reply:
Done